src/StackAdmin/Azs.Subscriptions.Admin/Module/Azs.Subscriptions.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Move-AzsSubscription.ps1

```diff
@@ -0,0 +1,147 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+#>
+
+<#
+.SYNOPSIS
+	
+
+.DESCRIPTION
+	Move subscriptions between delegated provider offers. This process will only perform a rebranding,
+	the underlying offer, plans, quotas for the subscriptions will not be altered.
+
+.PARAMETER DestinationDelegatedProviderOffer
+	Specifies the fully qualified delegated provider offer into which this cmdlet moves subscriptions.
+	NULL if the subscriptions are to be moved back to the Default Provider.
+
+.PARAMETER ResourceId
+	Specifies an array of fully qualified subscription resource identifiers that this
+	cmdlet moves.
+
+.PARAMETER AsJob
+	Specifies whether the move operation is to be executed as a job.
+
+.EXAMPLE
+	Move user subscriptions to a delegated provider offer.
+	Move-AzsSubscription `
+		-DestinationDelegatedProviderOffer "/subscriptions/45ec4d39-8dea-4d26-a373-c176ec53717a/providers/Microsoft.Subscriptions.Admin/delegatedProviders/798568b7-c6f1-4bf7-bb8f-2c8bebc7c777/offers/ro1"
+		-ResourceId "/subscriptions/45ec4d39-8dea-4d26-a373-c176ec53717a/providers/Microsoft.Subscriptions.Admin/subscriptions/ce4c7fdb-5a38-46f5-8bbc-b8b328a87ab6","/subscriptions/45ec4d39-8dea-4d26-a373-c176ec53717a/providers/Microsoft.Subscriptions.Admin/subscriptions/a0d1a71c-0b27-4e73-abfc-169512576f7d"
+
+.EXAMPLE
+	Move user subscriptions from a delegated provider to the Default Provider.
+	$resourceIds = Get-AzsUserSubscription -Filter "offerName eq 'o1'" | where {$_.DelegatedProviderSubscriptionId -eq "798568b7-c6f1-4bf7-bb8f-2c8bebc7c777"} | Select -ExpandProperty Id
+	Move-AzsSubscription -ResourceId $resourceIds
+#>
+function Move-AzsSubscription
+{
+	[CmdletBinding(DefaultParameterSetName='Subscriptions_MoveSubscriptions')]
+	param(    
+		[Parameter(Mandatory = $false)]
```

Comment:
Is this not Mandatory?

Reply:
Correct, null defines a removal of delegated provider offer.